executor/ddl.go

```diff
@@ -200,10 +203,65 @@ func (e *DDLExec) executeAlterDatabase(s *ast.AlterDatabaseStmt) error {
 }
 
 func (e *DDLExec) executeCreateTable(s *ast.CreateTableStmt) error {
+	if s.TemporaryKeyword == ast.TemporaryLocal {
+		return e.createSessionTemporaryTable(s)
+	}
+
 	err := domain.GetDomain(e.ctx).DDL().CreateTable(e.ctx, s)
 	return err
 }
 
+func (e *DDLExec) createSessionTemporaryTable(s *ast.CreateTableStmt) error {
+	is := e.ctx.GetInfoSchema().(infoschema.InfoSchema)
+	dbInfo, ok := is.SchemaByName(s.Table.Schema)
+	if !ok {
+		return infoschema.ErrDatabaseNotExists.GenWithStackByArgs(s.Table.Schema.O)
+	}
+	tbInfo, err := ddl.BuildTableInfoWithCheck(e.ctx, s, dbInfo.Charset, dbInfo.Collate)
```

Comment:
Does  BuildTableInfoWithCheck support `create table like ...` ?

Reply:
I'm not sure, `create table like ...` will be handle in some other PR, this one focus on creating a table.

Reply:
I checked the function ddl.CreateTable, ddl.CreateTableWithInfo, most codes can be reused here. We can just return if local temporary if founded before creating table in tikv. Current modification is easy to lost some checks. For example,
`create temporary table if not exists` will still return error because you didn't check `IfNotExists` in statement